daemon/service-db.cc

```diff
@@ -695,7 +696,10 @@ MLServiceDB::delete_model (const std::string name, const guint version)
   }
 
   if (version > 0U) {
-    if (is_model_activated (key_with_prefix, version))
+    if (force)
+      ml_logw ("Warnning:: The model with name %s and version %d is activated. But delete it.",
```

Comment:
Typo: we cannot sure the model is activated here, I suggest updating warning message here.
`The model with name %s and version %u may be activated, delete it from ml-service.`

Reply:
thanks, i update it.